tools/pkg/git/git.toit

```diff
@@ -174,7 +176,7 @@ class GitProtocol_:
         lines.add length
         continue
       if not buffer.try-ensure-buffered length:
-        throw "Premature end of input"
+        throw "Premature end of input."
```

Comment:
Thrown strings typically do not end with dot.

```suggestion
        throw "Premature end of input"
```

Reply:
done.